File: CPP/Baju.cpp
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris
{
private:
  string size, merk;

public:
  Baju(int id, string nama, int harga, int stok, string jenis, string bahan, string warna, string size, string merk)
  {
    this->setId(id);
    this->setNama(nama);
    this->setHarga(harga);
    this->setStok(stok);
    this->setJenis(jenis);
    this->setBahan(bahan);
    this->setWarna(warna);
    this->size = size;
    this->merk = merk;
  }

  void setSize(string size)
  {
    this->size = size;
  }

  void setMerk(string merk)
  {
    this->merk = merk;
  }

  string getSize()
  {
    return this->size;
  }

  string getMerk()
  {
    return this->merk;
  }

  // void addBaju(int id, string nama, int harga, int stok, string jenis, string bahan, string warna, string size, string merk) {
  //   Baju baju(id, nama, harga, stok, jenis, bahan, warna, size, merk);
  //   daftarBarang.push_back(baju);
  // }
};

File: PHP/Baju.php
<?php
require_once 'Aksesoris.php';

class Baju extends Aksesoris
{
  private $size = "";
  private $merk = "";

  public function __construct($id, $nama, $harga, $stok, $jenis, $bahan, $warna, $size, $merk)
  {
    parent::__construct($id, $nama, $harga, $stok, $jenis, $bahan, $warna);
    $this->size = $size;
    $this->merk = $merk;
  }

  public function getSize()
  {
    return $this->size;
  }

  public function getMerk()
  {
    return $this->merk;
  }

  public function setSize($size)
  {
    $this->size = $size;
  }

  public function setMerk($merk)
  {
    $this->merk = $merk;
  }
}
?>